Tests/PureeTests/LoggerTests.swift

```diff
@@ -84,6 +84,53 @@ class LoggerTests: XCTestCase {
         XCTAssertEqual(buffer.logs(for: "pv.*").count, 1)
     }
 
+    func testLoggerWithCustomSetting() {
+        struct CustomFilterSetting: FilterSettingProtocol {
+            private let tableName: String
+
+            init(tableName: String) {
+                self.tableName = tableName
+            }
+
+            func makeFilter() throws -> Filter {
+                return PVLogFilter(tagPattern: TagPattern(string: "pv2")!, options: ["table_name": tableName])
+            }
+        }
+
+        struct CustomOutputSetting: OutputSettingProtocol {
+            private let tableName: String
+
+            init(tableName: String) {
+                self.tableName = tableName
+            }
+
+            func makeOutput(_ logStore: LogStore) throws -> Output {
+                return PVLogOutput(logStore: logStore, tagPattern: TagPattern(string: "pv2")!, options: ["table_name": tableName])
+            }
+        }
+
+        let configuration = Logger.Configuration(logStore: logStore,
+                                                 dateProvider: DefaultDateProvider(),
+                                                 filterSettings: [
+                                                    FilterSetting(PVLogFilter.self, tagPattern: TagPattern(string: "pv")!),
+                                                    CustomFilterSetting(tableName: "pv_log"),
+                                                    FilterSetting(PVLogFilter.self, tagPattern: TagPattern(string: "pv.*")!)
```

Comment:
[nits] `,` at the end of the line.

Reply:
https://github.com/cookpad/Puree-Swift/pull/1/commits/b261a42a26a8a231c0b4af5ff498458fa56b9cf4

I added `,` and enable `mandatory_comma`.